### PR TITLE
feature: Add support for leaderboard archives

### PR DESCRIPTION
**Pull request by kirre-bylund**

### CODE REVIEW

File: LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerForCpp.cpp [hunk 1]
```diff
@@ -74,6 +74,16 @@ void ULootLockerServerForCpp::GetPaginatedScoresFromLeaderboard(FString Leaderbo
 	ULootLockerServerLeaderboardRequest::GetPaginatedScoresFromLeaderboard(LeaderboardKey, Count, After, FLootLockerServerGetScoresFromLeaderboardResponseBP(), OnCompletedRequest);
 }
 
+void ULootLockerServerForCpp::ListLeaderboardArchive(const FString& LeaderboardKey,	const FLootLockerServerLeaderboardArchiveResponseDelegate& OnCompletedRequest)
```
Comment (JohannesLoot) [inline LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerForCpp.cpp:77]: `const FLootLockerServerLeaderboardArchiveResponseDelegate& `
Spaces/tabs formatting

File: LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	FString metadata;
+	// The Player's member ID on the Archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	FString member_id;
+	// The Player's rank on the archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	int32 rank = 0;
+	// The Player's Score on the archived Leaderboard.
+	UPROPERTY(BlueprintReadWrite, EditAnywhere, Category = "LootLockerServer")
+	int score = 0;
```
Comment (JohannesLoot) [inline LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h:46]: Intentional that this an in ant not an int32?

Comment (kirre-bylund) [reply LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h:46]: Well, no. Or rather, the int32 should be just int :P

File: LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerForCpp.cpp (hunk 1 above)
Comment (kirre-bylund) [reply LootLockerServerSDK/Source/LootLockerServerSDK/Private/LootLockerServerForCpp.cpp:77]: Fixed

File: LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h (hunk 2 above)
Comment (kirre-bylund) [reply LootLockerServerSDK/Source/LootLockerServerSDK/Public/ServerAPI/LootLockerServerLeaderboardArchiveRequestHandler.h:46]: Fixed

**Pull request merged**